.github/workflows/rock-release-dev.yaml

```diff
@@ -36,18 +36,25 @@ jobs:
         sudo snap install --classic --channel edge rockcraft
 
     - name: Build ROCK
+      id: builder
       run: |
         rockcraft_dir="${{ steps.find-latest.outputs.latest-dir }}"
         current_wd=$(pwd) && cd $rockcraft_dir
         rockcraft pack --verbose
+        version=$(yq -r .version rockcraft.yaml)
+        echo "version=${version}" >> "$GITHUB_OUTPUT"
+        
         cd $current_wd
         cp $rockcraft_dir/${{ inputs.rock-name }}_*.rock $current_wd
         digest=$(skopeo inspect oci-archive:$(realpath ./${{ inputs.rock-name }}_*.rock) --format '{{.Digest}}')
         echo "digest=${digest#*:}" >> "$GITHUB_OUTPUT"
 
     - name: Upload ROCK to ghcr.io
       run: |
+        # dev tag
         sudo skopeo --insecure-policy copy oci-archive:$(realpath ./${{ inputs.rock-name }}_*.rock) docker://ghcr.io/canonical/${{ inputs.rock-name }}:dev --dest-creds "observability-noctua-bot:${{ secrets.OBSERVABILITY_NOCTUA_TOKEN }}"
+        # version tag
+        sudo skopeo --insecure-policy copy oci-archive:$(realpath ./${{ inputs.rock-name }}_*.rock) docker://ghcr.io/canonical/${{ inputs.rock-name }}:${{ steps.builder.outputs.version }} --dest-creds "observability-noctua-bot:${{ secrets.OBSERVABILITY_NOCTUA_TOKEN }}"
```

Comment:
Seems like [`--additional-tag`](https://github.com/containers/skopeo/blob/main/docs/skopeo-copy.1.md) may not be supported by ghcr.
Must be a better way to add a second tag without skopeo copy again?

Reply:
No idea, that's what we were doing before